js-tests/build/specs.js

```diff
@@ -180,6 +180,23 @@ describe( 'Shortcode View Constructor', function(){
 		var shortcode = ShortcodeViewConstructor.parseShortcodeString( '[no_inner_content foo="bar"]burrito[/no_inner_content]' );
 		var _shortcode = $.extend( true, {}, shortcode );
 		expect( _shortcode.formatShortcode() ).toEqual( '[no_inner_content foo="bar"]burrito[/no_inner_content]' );
+		ShortcodeViewConstructor.shortcode = {
+			'type' : 'single',
+			'tag' : 'no_inner_content',
+			'attrs' : {
+				'named' : {
+					'foo' : 'bar',
+				},
+				'numeric' : [],
+			},
+			'content' : 'burrito'
+		};
+		var ShortcodeViewConstructorWithoutFetch = ShortcodeViewConstructor;
+		ShortcodeViewConstructorWithoutFetch.delayedFetch = function() {
+			return new $.Deferred();
+		};
```

Comment:
> Why the extra variable assignment here?

Because I don't want to modify the original? Although, I see below that I'm still using the original in the test...